packages/react/src/experimental/Widgets/Content/ListItems/WidgetAvatarsListItem/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { cn } from "../../../../../lib/utils"
+import { AvatarVariant } from "../../../../Information/Avatars/Avatar"
+import { AvatarList } from "../../../../Information/Avatars/AvatarList/index.tsx"
+import { EmojiAvatar } from "../../../../Information/Avatars/EmojiAvatar"
+
+type Props<Id extends string | number = string | number> = {
+  id: Id
+  emoji: string
+  title: string
+  subtitle?: string
+  avatars: AvatarVariant[]
+  onClick?: (id: Id) => void
+}
+
+export type WidgetAvatarsListItemProps<
+  Id extends string | number = string | number,
+> = Props<Id>
+
+type WrapperProps = {
+  onClick?: (ev: React.MouseEvent<HTMLAnchorElement>) => void
+  className: string
+  children: React.ReactNode
+}
+
+const Wrapper: React.FC<WrapperProps> = ({ onClick, className, children }) => {
+  return onClick ? (
+    <a className={className} onClick={onClick} tabIndex={0}>
+      {children}
+    </a>
+  ) : (
+    <div className={className} tabIndex={-1}>
```

Comment:
Not needed, div is not an interactive element by default
```suggestion
    <div className={className}>
```